boards/shields/nrf5340_audio_display/nrf5340_audio_display.overlay

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 PHYTEC Messtechnik GmbH
```

Comment:
Do we need all of this for the overlay? It should be enough to define some pins, SPI speed etc.

Reply:
Most of the numbers for the screen are a bit magic, even after reading the documentation. They're copied directly from the Zephyr generic overlay, and changing some of them causes the screen to not work, and others just changes the color scheme. Philip did testing for all of them and concluded they were necessary.